TrashMob/client-app/src/components/Customization/HeroSection.tsx

```diff
@@ -1,26 +1,39 @@
-import { FC } from 'react';
-import { Col, Container, Image, Row } from 'react-bootstrap';
-import globes from '../assets/gettingStarted/globes.png';
+import { FC } from "react";
+import { Col, Image, Row } from "react-bootstrap";
+import globes from "../assets/gettingStarted/globes.png";
 
 interface HeroSectionProps {
-    Title: string;
-    Description: string;
-};
+  Title: string;
+  Description: string;
+}
 
 export const HeroSection: FC<HeroSectionProps> = ({ Title, Description }) => {
-    return (
-        <Container fluid className='bg-grass'>
-            <div className='container mx-auto'>
-                <Row className="text-left pt-0">
-                    <Col className="d-flex flex-column justify-content-center pr-5">
-                        <h1 className="font-weight-bold">{Title}</h1>
-                        <p className="font-weight-bold">{Description}</p>
-                    </Col>
-                    <Col>
-                        <Image src={globes} alt="globes" className="h-100 mt-0" />
-                    </Col>
-                </Row>
-            </div>
-        </Container>
-    )
-}
\ No newline at end of file
+  return (
+    <div className="bg-grass ">
+      <div className="container-lg mx-auto">
```

Comment:
I am not a CSS person, but should we consider moving toward package agnostic classnames?

Reply:
Could you elaborate a little bit? These classnames are bootstrap utility classes, so it's not really feasible to change the class names unless we're jumping out of bootstrap into pure CSS. What was discussed was eventually moving over to Tailwind, which has similarly named classes (but different enough that the migration will be a pain.)

Reply:
I think I'm speaking broadly of implementing a design system to remove our dependency on a UI package. Its a conversation for another time I guess.